gr-dtv/lib/atsc/atsc_fs_checker_impl.cc

```diff
@@ -69,7 +69,7 @@ int atsc_fs_checker_impl::general_work(int noutput_items,
         for (int j = 0; j < LENGTH_511 && errors < PN511_ERROR_LIMIT; j++)
             errors += (in[i].data[j + OFFSET_511] >= 0) ^ atsc_pn511[j];
 
-        // std::cout << errors << std::endl;
```

Comment:
This is not quite the same change here.

Reply:
yeah, this should become a lower-prio log